docs/iss/2026/index.md

```diff
@@ -0,0 +1,162 @@
+<style>
+.md-content .md-typeset h1 {
+    display: none;
+}
+.table {
+    display: table;
+}
+.row {
+    display: table-row;
+}
+.cell {
+    display: table-cell;
+    padding: 2px;
+    padding-right: 20px;
+}
+</style>
+
+<figure markdown="span">
+    <img src="../../assets/iss2026.png" alt="drawing" width="70%"/>
+    <figcaption>**Maintaining the Joy of Software Development**</figcaption>
+</figure>
+
+<div class="table">
+    <div class="row">
+        <div class="cell"><b>When:</b></div>
+        <div class="cell">6-10 April, 2026</div>
+    </div>
+    <div class="row">
+        <div class="cell"><b>Where:</b></div>
+        <div class="cell">Boulder, CO and virtual</div>
+    </div>
+</div>
+
+The annual Improving Scientific Software Conference brings together software
+engineers, scientists, developers - any individuals with an interest in
+scientific software design - to share novel experiences and best practices,
+develop connections across divisions and institutions, and advance our
+community.
+
+The scientific software ecosystem is experiencing changes unlike any in
+recent times. Large language models are revolutionizing the software design
+process, but understanding how to use them effectively and ethically remains a
+major difficulty. Software support is more uncertain than ever, given the
+disruptions we've seen to previously reliable funding streams, necessitating
+more collaboration between public and private sectors than ever before. As
+always, these challenges bring with them corresponding opportunities, and in
+pursuing such opportunities, we can seek to *maintain the joy of software
+development*.
```

Comment:
This section feels a bit prescriptive to me.  Would folks be open to stopping at framing some of the challenges we're facing and leaving potential solutions and opportunities to be posed as ideas shared during the conference?

Happy to have this conversation elsewhere if that's better / more comfortable.

Reply:
I've pushed a revised version that strips the suggestions out. I think your point about it being too prescriptive is logical.